components/onboarding_tasklist/onboarding_tasklist_completed.tsx

```diff
@@ -150,13 +160,12 @@ const Completed = (props: Props): JSX.Element => {
                                     id='onboardingTask.checklist.start_enterprise_now'
                                     defaultMessage='Start your free Enterprise trial now!'
```

Comment:
I believe the latest message for is `Learn More...` can you double check the Figma for this?

Reply:
Nvm, I see this is for the completed step now.